jasmin_services/models/access_control.py

```diff
@@ -30,16 +52,13 @@ def _active_pks(self):
         """
         Returns a list of the primary keys of the active grants.
         """
-        # Here, we utilise the fact that we have an auto-incrementing id field
-        # to get the largest id for each role/user combo
-        # This uses DISTINCT ON, so is **Postgres specific**
         # We use all the grants, rather than self, because this queryset might
         # have filters applied that will mean we identify the wrong objects as
         # active
```

Comment:
The logic in this comment doesn't actually apply anymore. It is totally fine to just filter the current query using `next_grant__isnull = True`. We can probably do away with `_active_pks` and just filter the current query in `filter_active`.